### PR TITLE
Feature: Added properties option when right clicking recent files

### LINKED ISSUE #11434: Feature: Add properties option when right clicking recent files
### What feature or improvement do you think would benefit Files?

Add properties option when right clicking recent files

### Requirements

- Add properties option when right clicking recent files

### Files Version

v2.4.40

### Windows Version

Windows 11

### Comments

_No response_

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #11434 

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

**Comment by QuaintMako** [comment]
I am managing to get a crash when playing around with the code, that is not happening all the time though:

1- Open the property window of a recent file and open the detail panel.
2- Open a second property window of the same file, open the detail panel.
3- Try to close the first window. It may lead to a crash.

**Comment by hishitetsu** [comment]
> I'll try get the log files later today, and try to see if I can reproduce it when opening properties a file that is not in the recent files list;

I can reproduce it and it also occurs when opening properties outside of the recent files list.
I think this is a serious issue, but it is a separate issue from this PR.

**Review by QuaintMako — APPROVED** [review]

**Comment by hishitetsu** [comment]
@QuaintMako no, can you please open an issue?

File: src/Files.App/Views/HomePage.xaml.cs [hunk 1]
```diff
@@ -121,6 +121,7 @@ public void ReloadWidgets()
 			{
 				Widgets.ViewModel.InsertWidget(new(recentFilesWidget, (value) => UserSettingsService.PreferencesSettingsService.RecentFilesWidgetExpanded = value, () => UserSettingsService.PreferencesSettingsService.RecentFilesWidgetExpanded), 4);
 
+				recentFilesWidget.AppInstance = AppInstance;
```
Comment (yaira2) [inline src/Files.App/Views/HomePage.xaml.cs:124]: This looks like the cause of #12160 🤔

Comment (hishitetsu) [reply src/Files.App/Views/HomePage.xaml.cs:124]: I tested just after this PR was merged and the CPU usage is not high.
image

**Pull request merged**